### DISCUSSION TITLE
Web UI for monitor distributed progress

### DISCUSSION THREAD

**Discussion by bohutang** [databendlabs/databend#6402] [Ideas]
**Summary**
Web UI for monitor distributed progress, like Spark UI.

**Comment by tlightsky** [comment]
any preferred UI framework or libraries?

**Comment by bohutang** [comment]
Not yet, feel free to recommend, beautiful is better :)

**Comment by tlightsky** [comment]
here are some points just come to my mind:
* it should be embed to fusequery/fusestore binary so that user can visit it easily(like [spark ui](https://github.com/apache/spark/blob/0494dc90af48ce7da0625485a4dc6917a244d580/core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala))
* [yew](https://github.com/yewstack/yew) is an option to use both rust in frontend and backend (could use same definition of some structs)
* the main function is show list and graph from server metrics to make current status more visible and easy to manage

**Comment by PsiACE** [comment]
FYI, https://github.com/stateright/stateright

**Comment by Xuanwo** [comment]
I believe this issue is out of the scope of databend and not actionable. 

How about closing it now? @BohuTANG 

Or, we can convert this issue to the discussion as new ideas.
